app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -98,6 +98,21 @@ def handle_standard_error(err, enrollment)
       def analytics(user: AnonymousUser.new)
         Analytics.new(user: user, request: nil, session: {}, sp: nil)
       end
+
+      def transliterate(value)
+        return value unless IdentityConfig.store.usps_ipp_transliteration_enabled
+
+        result = transliterator.transliterate(value)
+        if result.unsupported_chars.size > 0
+          result.original
+        else
+          result.transliterated
+        end
```

Comment:
1. this logic seems backwards? if there are *unsupported characters* then we want to return the transliterated version right?
2. could we use the `#changed?` property here?
```suggestion
        if result.changed?
          result.transliterated
        else
          result.original
        end
```

Reply:
No, this logic is correct. `unsupported_chars` contains characters that we cannot transliterate.